client-v2/src/components/FeedsContainer.tsx

```diff
@@ -239,93 +272,124 @@ class FeedsContainer extends React.Component<
-              </select>
-            </SortByContainer>
-          </Sorters>
-        </ResultsHeadingContainer>
-      </FixedContentContainer>
+      <React.Fragment>
+        <SearchInput
+          updateDisplaySearchFilters={this.updateDisplaySearchFilters}
+          displaySearchFilters={this.state.displaySearchFilters}
+          onUpdate={this.handleParamsUpdate}
+          showReviewSearch={false}
+          rightHandContainer={<ClipboardHeader />}
+        />
+        {!displaySearchFilters && (
+          <FixedContentContainer>
+            <ResultsHeadingContainer>
+              <div>
+                <Title>
+                  Latest
+                  <ShortVerticalPinline />
+                </Title>
+                <RefreshButton
+                  disabled={this.isLoading}
+                  onClick={() => this.runSearchAndRestartPolling()}
+                >
+                  {this.isLoading ? 'Loading' : 'Refresh'}
+                </RefreshButton>
+              </div>
+              <Sorters>
+                <TopOptions>
+                  <RadioGroup>
+                    <RadioButton
+                      checked={this.state.capiFeedIndex === 0}
+                      onChange={() => this.handleFeedClick(0)}
+                      label="Live"
+                      inline
+                      name="capiFeed"
+                    />
+                    <RadioButton
+                      checked={this.state.capiFeedIndex === 1}
+                      onChange={() => this.handleFeedClick(1)}
+                      label="Draft"
+                      inline
+                      name="capiFeed"
+                    />
+                  </RadioGroup>
+                  {pagination && hasPages && (
+                    <PaginationContainer>
+                      <Pagination
+                        pageChange={this.pageChange}
+                        currentPage={pagination.currentPage}
+                        totalPages={pagination.totalPages}
+                      />
+                    </PaginationContainer>
+                  )}
+                </TopOptions>
+                <SortByContainer>
+                  <label htmlFor="sort-results">Sort by:</label>
+                  <select
+                    id="sort-results"
+                    onChange={this.sortResultsBy}
+                    value={this.state.sortByParam}
+                  >
+                    <option value="first-publication">First published</option>
+                    <option value="published">Latest published</option>
+                  </select>
+                </SortByContainer>
+              </Sorters>
+            </ResultsHeadingContainer>
+          </FixedContentContainer>
+        )}
+      </React.Fragment>
     );
   };
 
   public render() {
     const {
+      isPrefillMode,
       liveArticles,
       previewArticles,
       liveError,
-      previewError
+      previewError,
```

Comment:
As above re: reasoning about the source of this stuff. It'd be cleaner to select this out if we could.